static/src/javascripts/bootstraps/enhanced/common.js

```diff
@@ -267,6 +268,7 @@ const initialiseEmail = (): void => {
 const initialiseBanner = (): void => {
     // ordered by priority
     const bannerList = [
+        firstPvConsentPlusEngagementBanner,
```

Comment:
I think this changes the behaviour of the engagement banner, previously it would only come up after 3 pageviews, but now it comes up on the first pageview. Was this intentional?

Reply:
I don't think it does change this, the `canShow` for `firstPvConsentPlusEngagementBanner` calls the `canShow ` of the `membershipEngagementBanner` which still contains the view count check.